Taskfile.yml

```
@@ -126,3 +126,19 @@ tasks:
   general:prepare-deps:
     desc: Prepare project dependencies for license check
     # No preparation is needed for Go module-based projects.
+
+  protoc:compile:
+    desc: Compile protobuf definitions
+    cmds:
+      - buf dep update
```

I would install buf as a go tool. You can run `go get --tool github.com/bufbuild/buf` to add the latest buf version in the go.mod, and then here you can write

```suggestion
      - go tool buf dep update
```